tests/Unit/CssInlinerTest.php

```diff
@@ -1606,13 +1606,18 @@ public function orderedRulesAndSurroundingCssDataProvider()
                 foreach ($possibleSurroundingCss as $descriptionAfter => $cssAfter) {
                     // every combination would be a ridiculous c.1000 datasets - choose a select few
                     // test all possible CSS before once
-                    if (($cssBetween === '' && $cssAfter === '')
+                    if (
+                        ($cssBetween === ''
+                        && $cssAfter === '')
```

Comment:
... I find having a newline within the bracketed `AND` expressions makes the code less readable, not more.  Is this really not allowed?